app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+
+    for i in phrase:
+        if i.lower() == letter.lower():
+            count += 1
+    return count
```

Comment:
You could try to implement this task in one line using built-in string methods and chaining to not allocate additional variables throughout your code execution.